handbook/brand.md

```diff
@@ -207,22 +207,22 @@ In Figma:
   * Avoid using SVGs or icon fonts.
 3. Click the __Export__ button.
 
-## Brand rituals
+## Rituals
 
-The following rituals are engaged in by the  directly responsible individual (DRI) and at the frequency specified for the ritual.
+Directly Responsible Individuals (DRI) engage in the rituals below at the frequency specified.
 
 | Ritual                       | Frequency                | Description                                         | DRI               |
 |:-----------------------------|:-----------------------------|:----------------------------------------------------|-------------------|
-| Documentation quality | Daily | Review pull requests for tone and brand consistency | Mike Thomas |
-| Tweet review | Daily | Review tweets for tone and brand consistency | Mike Thomas |
-| Community Slack  | Daily   | Check Fleet and osquery Slack channels for community questions, ensure questions are responded to and logged | Kathy Satterlee |
-| Social media check-in     |  Daily | Check social media for community questions and ensure they are responded to.  Generate dev advocacy-related content | Kelvin Omereshone   |
+| Documentation quality | Daily | Review pull requests for tone and brand consistency. | Mike Thomas |
+| Tweet review | Daily | Review tweets for tone and brand consistency. | Mike Thomas |
+| Community Slack  | Daily   | Check Fleet and osquery Slack channels for community questions and ensure questions are responded to and logged. | Kathy Satterlee |
```

Comment:
```suggestion
| Community Slack  | Daily   | Check Fleet and osquery Slack channels for community questions and make sure questions are responded to and logged. | Kathy Satterlee |
```
I keep meaning to add "ensure" to our rules. We can Mr. Rogers it by making sure that we always use "make sure" rather than "ensure."